### ISSUE TITLE
Demo Map - Show All Layers in a Single Map Interface

### ISSUE THREAD

**Issue by allella** [codeforgreenville/OpenData#34]

**Comment by allella** [comment]
Consider how we can syndicate data already published by folks like
http://greenvilleopenmap.info/

and figure out how to get some of our community SRT data layers on these maps if they are interested.  And/or invite Greenville County Rec to edit the spreadsheets we use.

@bikeoid (Mike Nice) has http://greenvilleopenmap.info/ where we might show a preview off all the real-time map layers.

**Comment by allella** [comment]
Partial SRT demo map https://data.openupstate.org/maps/demo/

This demo doesn't have all the layers at https://data.openupstate.org/map-layers#swamp-rabbit-trail-layers so all those layers could be used. Jim can share the Leaflet code from this demo to show how to do multiple layers.

**Comment by bikeoid** [comment]
There's also this curated dataset from the county, partnering with GHS https://www.unitedwaygc.org/imap ,  article with more details at http://www.greenvilleonline.com/story/news/local/2017/06/21/county-partners-furman-ghs-bring-resources-you/415259001/

**Comment by allella** [comment]
Thanks Mike.  Pam also made the IMAP Furman connection and we intend to work with them in some capacity.

**Comment by allella** [comment]
@Jaaron0606 
iMAP Project - maintained by GHS, Greenville County, Furman, United Way
http://www.gcgis.org/apps/imap/

County REST API - https://www.gcgis.org/arcgis/rest/services/imap/imap/MapServer/70

**Comment by allella** [comment]
We could eventually setup a time with Rhett Baker and Mike Winiski at Furman to talk about better reuse and syndication of county map layers.

For instance, the parks and Swamp Rabbit trail layers are probably better coming from the county as an origin and all we do with those layers is provide them in different formats and make it easier for other people to consume and trust a layer.

**Comment by allella** [comment]
@SpaceCowboy326 has a good start on a prettier multi-layer map. Pulling him into the conversation.

**Comment by SpaceCowboy326** [comment]
Thanks @allella, it's pretty ugly at the moment but I'm hoping to style it a bit soon.  The repository can currently be found at [https://github.com/SpaceCowboy326/open_data_gvl](url) for anyone who is interested.

**Comment by allella** [comment]
I thought it looked like progress over that demos we have so keep going.   If you get a live URL where folks can preview the map itself then that would likely get more feedback than just the source code, but all is appreciate.

**Comment by allella** [comment]
Tim @SpaceCowboy326 wins member of the month for his work on the demo of all map layers.

**Comment by SpaceCowboy326** [comment]
Haha, thanks @allella!  I will try to get the demo running consistently on my web server tonight so anyone who is interested can check it out.

**Comment by SpaceCowboy326** [comment]
Still working on a more appropriate domain name (this was meant for a personal project) and cleaning up some of the code, but the map layer demo can currently be viewed at https://yubnub.net

**Comment by SpaceCowboy326** [comment]
@allella hey Jim - sorry about that, I was trying to figure out how to run multiple apps from my hosting service but ended up having to run one at a time.  I've set the map application back up so it should still be available at yubnub.net.

I can't say that I have anything in mind for further development at the moment, but if there was anything in particular you'd like to see I'm open to suggestions (I was having trouble trying to figure out next steps and ended up just putting the project down for a while).

**Comment by allella** [comment]
Alright. At this point it may make the most sense if we host the map under a subdomain on the data.openupstate.org for public view until there's more development plans and definition on a dedicated domain.

**Comment by SpaceCowboy326** [comment]
@allella sorry for my slow responsiveness.  I am serving the map demo on my site again at yubnub.net, but would be happy to have you guys serve it from data.openstate.org.  The code is still hosted here if you want to clone it:
https://github.com/SpaceCowboy326/open_data_gvl

However, I think the script that pulls the locations may currently be broken.  I will try to get that working the way I wanted it to or revert it to the last working state by the end of this week. I'll message you when it's all set, then I'm happy to help get it up and running some time.

**Comment by allella** [comment]
Thanks for the update.

**Comment by SpaceCowboy326** [comment]
@allella again, apologies for leaving this unresolved for so long, but I've made some (mostly cosmetic) updates to the app and tried to clean it up a bit. The code is still messy and has a lot of TODOs, but If you're still interested in hosting it I would be happy to try and help get it running somewhere.

**Comment by allella** [comment]
Sure, we can host.  Is the README sufficient to get things going, or is there more that's needed?

Thanks,
Jim

**Comment by SpaceCowboy326** [comment]
I think that should do it, I've installed on a few different machines and haven't had much trouble getting it running.  Let me know here or on https://github.com/SpaceCowboy326/open_data_gvl if you run into issues.

**Comment by allella** [comment]
This project is currently up https://data.openupstate.org/map/preview/all-layers/dist/#/

but I'm going to move it to another sub-directory.

Thanks @SpaceCowboy326

**Comment by SpaceCowboy326** [comment]
Awesome, thanks @alella! I'll let you know if I get any new features working that are worth updating for, always open to suggestions as well.

**Comment by allella** [comment]
This is now linked to on https://data.openupstate.org/map-layers

I'd like to get rid of the /dist part of the URL but I can't find the time to do more pressing things, so I'll just deal with it and be glad we have it hosted.

I also forked and renamed the repo so it's easier to share with people from CFG. I kept having to look for it. https://github.com/codeforgreenville/open-map-data-multi-layers-demo

Thanks again.
